Framework/DataObjects/test/WorkspaceCreationTest.h

```diff
@@ -92,7 +92,7 @@ void run_create_partitioned_with_instrument(const Parallel::Communicator &comm,
   TS_ASSERT_EQUALS(i.size(), expectedSize);
 }
 
-void run_indexInfo_legacy_compatibility_partitioned_workspace_failure(const Parallel::Communicator &comm) {
```

Comment:
changes in this file should be reverted. `run_indexInfo` is conceptually unrelated to `MDEvent.runIndex`

Reply:
done!